src/ServicePulse.Host/vue/package.json

```diff
@@ -17,16 +22,29 @@
     "d3": "^7.6.1",
     "highlight.js": "^11.6.0",
     "moment": "^2.29.4",
-    "vue": "^3.2.41",
+    "pinia": "^2.1.7",
+    "vue": "^3.3.4",
     "vue-highlight-code": "^0.1.16",
-    "vue-router": "^4.1.5",
+    "vue-router": "^4.2.5",
     "vue-toastification": "^2.0.0-rc.5",
     "vue3-cookies": "^1.0.6",
     "vue3-simple-typeahead": "^1.0.11"
   },
   "devDependencies": {
     "@rushstack/eslint-patch": "^1.7.0",
+    "@storybook/addon-a11y": "^7.5.3",
+    "@storybook/addon-actions": "^7.6.2",
+    "@storybook/addon-essentials": "^7.5.3",
+    "@storybook/addon-interactions": "^7.6.5",
+    "@storybook/addon-links": "^7.5.3",
+    "@storybook/blocks": "^7.5.3",
+    "@storybook/jest": "^0.2.3",
+    "@storybook/test-runner": "^0.14.1",
+    "@storybook/testing-library": "^0.2.2",
+    "@storybook/vue3": "^7.5.3",
+    "@storybook/vue3-vite": "^7.5.3",
```

Comment:
Should these be in here? We had a separate PR to remove the testing and were planning on re-introducing it.

Reply:
Good catch. I'll take care of removing these an others in this file in the meantime